### ISSUE TITLE
ListItem menu not mobile accessible

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/nextcloud-vue#2339]
You cannot access the menu of a ListItem on mobile. This is a huge regression.
If the entry has a link, you have to long-press, cancel the browser's menu and if you're lucky, the entry will be "focused" and the menu shown.

@jancborchardt @marcoambrosini
